tests/files/gce_coreos-vault.yml

```diff
@@ -0,0 +1,13 @@
+# Instance settings
+cloud_machine_type: "n1-standard-2"
+cloud_image_family: coreos-stable
+cloud_region: us-central1-b
+mode: aio
+
+# Instance settings
+cert_management: vault
```

Comment:
Vault was removed, so we should probably remove this.

Reply:
already remove.

Reply:
Should probably rename this test file as well.

Reply:
delete it